cmd/tink-cli/cmd/template/create.go

```diff
@@ -12,21 +13,25 @@ import (
 	"github.com/spf13/cobra"
 	"github.com/tinkerbell/tink/client"
 	"github.com/tinkerbell/tink/protos/template"
+	"gopkg.in/yaml.v2"
 )
 
 var (
 	fPath        = "path"
-	fName        = "name"
 	filePath     string
 	templateName string
 )
 
+type TemplateName struct {
+	Name string `yaml:"name"`
+}
+
 // createCmd represents the create subcommand for template command
 var createCmd = &cobra.Command{
 	Use:   "create",
 	Short: "create a workflow template ",
 	Example: `tink template create [flags]
-cat /tmp/example.tmpl | tink template create -n example`,
+cat /tmp/example.tmpl | tink template create `,
```

Comment:
Can we please add more examples in the `Long` description?
For reference, you can check [`tink events watch`](https://github.com/tinkerbell/tink/blob/master/cmd/tink-cli/cmd/events/watch.go#L70).